schema/direction.mod

```diff
@@ -790,9 +790,13 @@
 	    Tristan
 	
 	The "other" kind is used when the harmony is entirely
-	composed of add elements. The "none" kind is used to
-	explicitly encode absence of chords or functional
-	harmony.
+	composed of add elements. 
+	
+	The "none" kind is used to explicitly encode absence of
+	chords or functional harmony. In this case, the root or
+	function element has no meaning. When using the root
+	element, the text attribute should be set to the empty

```

Comment:
I was confused here whether the `@text` attribute belongs to `root/root-step` or `kind`, since both have one. Would be good to mention `root-step` explicitly.

Reply:
Thanks for catching this. It should be fixed now.